etcdserver: leader changed during cluster creation causing test failures

It intermittently causes CI failures in the `integration-tests-container` test[0]

Unclear if etcd leader should change during cluster creation too.

[0]
```
time="2020-06-22T20:25:34Z" level=info msg="running ../../../cmd/wksctl/wksctl [apply --cluster=/tmp/multimaster_test_866680275/cluster.yaml --machines=/tmp/multimaster_test_866680275/machines.yaml --config-directory=/tmp/multimaster_test_866680275 --verbose --controller-image=docker.io/weaveworks/wksctl-controller:v0.8.3-rc.4]"
time="2020-06-22T20:27:23Z" level=info msg="running ../../../cmd/wksctl/wksctl [kubeconfig --cluster=/tmp/multimaster_test_866680275/cluster.yaml --machines=/tmp/multimaster_test_866680275/machines.yaml]"
time="2020-06-22T20:27:23Z" level=info msg="running kubectl [get nodes -o json --kubeconfig=]"
time="2020-06-22T20:27:24Z" level=info msg="The cluster currently has 1 node(s)"
time="2020-06-22T20:27:54Z" level=info msg="running kubectl [get nodes -o json --kubeconfig=]"
time="2020-06-22T20:27:54Z" level=info msg="The cluster currently has 1 node(s)"
time="2020-06-22T20:28:24Z" level=info msg="running kubectl [get nodes -o json --kubeconfig=]"
time="2020-06-22T20:28:24Z" level=info msg="The cluster currently has 1 node(s)"
time="2020-06-22T20:28:54Z" level=info msg="running kubectl [get nodes -o json --kubeconfig=]"
time="2020-06-22T20:28:54Z" level=info msg="The cluster currently has 1 node(s)"
time="2020-06-22T20:29:24Z" level=info msg="running kubectl [get nodes -o json --kubeconfig=]"
--- FAIL: TestMultimasterSetup (291.36s)
    multimaster_test.go:408: 
        	Error Trace:	multimaster_test.go:408
        	            				multimaster_test.go:392
        	            				multimaster_test.go:322
        	Error:      	Received unexpected error:
        	            	exit status 1
        	Test:       	TestMultimasterSetup
        	Messages:   	Command /usr/local/bin/kubectl failed. STDOUT: {
        	            	    "apiVersion": "v1",
        	            	    "items": [],
        	            	    "kind": "List",
        	            	    "metadata": {
        	            	        "resourceVersion": "",
        	            	        "selfLink": ""
        	            	    }
        	            	}
        	            	
        	            	STDERR: Error from server: rpc error: code = Unavailable desc = etcdserver: leader changed
FAIL
FAIL	github.com/weaveworks/wksctl/test/integration/container	291.363s
make: *** [integration-tests-container] Error 1

Exited with code exit status 2
CircleCI received exit code 2
```

--- a/test/integration/container/multimaster_test.go
+++ b/test/integration/container/multimaster_test.go
@@ -429,7 +429,11 @@ func TestMultimasterSetup(t *testing.T) {
 
 			var nodeList corev1.NodeList
 			for {
-				jsonOut := run(t, "kubectl", "get", "nodes", "-o", "json", fmt.Sprintf("--kubeconfig=%s", kubeconfig(out)))
+				jsonOut, stderr := doRun("kubectl", "get", "nodes", "-o", "json", fmt.Sprintf("--kubeconfig=%s", kubeconfig(out)))
+				if stderr != "" {
+					log.Warnf("error from kubectl; ignoring: %s", stderr)
+					continue
+				}
 				if err := json.Unmarshal([]byte(jsonOut), &nodeList); err != nil {
 					log.Warnf("Error deserialising output of kubectl get nodes: %s", err)
 				}

